packages/BetterPhpDocParser/Annotation/ChangeResolvedClassInParticularContextForAnnotation.php

```diff
@@ -18,26 +18,31 @@ class ChangeResolvedClassInParticularContextForAnnotation
         ],
     ];
 
-    public function __invoke(
+    public function changeResolvedClassIfNeed(
         AnnotationToAttribute $annotationToAttribute,
         DoctrineAnnotationTagValueNode $docNode
     ): void {
         foreach (self::RULES as $item) {
-            $this->applyRule($docNode, $item, $annotationToAttribute);
+            $this->applyRule(
+                $docNode,
+                new ChangeResolvedClassInParticularContextForAnnotationRule(
+                    $item['tag'],
+                    $item['value'],
+                    $item['resolved_class']
+                ),
```

Comment:
The type object wrapping should happen in constructor, as the values in `$item` are still weak array here.

Reply:
done